pkgs/by-name/im/imhex/package.nix

```diff
@@ -60,29 +62,33 @@ stdenv.mkDerivation rec {
     mbedtls
     nlohmann_json
     yara
+  ] ++ lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk_11_0.frameworks.UniformTypeIdentifiers
   ];
 
   cmakeFlags = [
-    "-DIMHEX_OFFLINE_BUILD=ON"
-    "-DUSE_SYSTEM_CAPSTONE=ON"
-    "-DUSE_SYSTEM_CURL=ON"
-    "-DUSE_SYSTEM_FMT=ON"
-    "-DUSE_SYSTEM_LLVM=ON"
-    "-DUSE_SYSTEM_NLOHMANN_JSON=ON"
-    "-DUSE_SYSTEM_YARA=ON"
+    (lib.cmakeBool "IMHEX_OFFLINE_BUILD" true)
+    (lib.cmakeBool "IMHEX_COMPRESS_DEBUG_INFO" false) # avoids error: cannot compress debug sections (zstd not enabled)
+    (lib.cmakeBool "IMHEX_GENERATE_PACKAGE" stdenv.isDarwin or stdenv.isWindows)
+    (lib.cmakeBool "USE_SYSTEM_CAPSTONE" true)
+    (lib.cmakeBool "USE_SYSTEM_CURL" true)
+    (lib.cmakeBool "USE_SYSTEM_FMT" true)
+    (lib.cmakeBool "USE_SYSTEM_LLVM" true)
+    (lib.cmakeBool "USE_SYSTEM_NLOHMANN_JSON" true)
+    (lib.cmakeBool "USE_SYSTEM_YARA" true)
   ];
 
-  # rsync is used here so we can not copy the _schema.json files
   postInstall = ''
     mkdir -p $out/share/imhex
-    rsync -av --exclude="*_schema.json" ${patterns_src}/{constants,encodings,includes,magic,patterns} $out/share/imhex
+    cp -aR ${patterns_src}/{constants,encodings,includes,magic,patterns} $out/share/imhex
+    find $out/share/imhex -name '*_schema.json' -exec rm {} \;
   '';
 
   meta = with lib; {
-    description = "Hex Editor for Reverse Engineers, Programmers and people who value their retinas when working at 3 AM";
+    description = "Hex Editor for Reverse Engineers, Programmers";
```

Comment:
I think this reads nicer 🙂 
```suggestion
    description = "Hex Editor for Reverse Engineers and Programmers";
```

Reply:
It seems it was copied from the official about
<img width="311" alt="image" src="https://github.com/user-attachments/assets/9a45e0b5-a0ec-490a-ad5c-ae186eaca7c7">
https://github.com/WerWolv/ImHex